plugin-build/CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+This is the source for the Module Graph Gradle plugin.
+
+# Formatting
+The code style of this project is enforced with Detekt. Run it with:
```

Comment:
Good idea to create one of these.

My recommended workflow is to navigate to the root and run:
- `./gradlew reformatAll` for autofixing formating issues
and 
- `./gradlew preMerge` for validating everything (this runs ktlint, detekt and tests)

Reply:
Nice, good to know! I've added this to the document